guides/project-service-management-guide.md

```diff
@@ -266,6 +266,10 @@ below:
   -g, --helm-config-git <helm-git>                            bedrock helm chart configuration git repository. --helm-chart-* and --helm-config-* are exclusive; you may only use one. (default: "")
   -b, --helm-config-branch <helm-branch>                      bedrock custom helm chart configuration branch. --helm-chart-* and --helm-config-* are exclusive; you may only use one. (default: "")
   -p, --helm-config-path <helm-path>                          bedrock custom helm chart configuration path. --helm-chart-* and --helm-config-* are exclusive; you may only use one. (default: "")
+  --service-build-vg <variable-group>                         existing azure devops variable groups
+  may use multiple. (default: "")
+  --service-build-variables <variables>                       existing variables from azure devops variable groups
+  may use multiple. (default: "")
```

Comment:
Document how to use multiple...for both CLI options

Reply:
Done